python/lsst/alert/packet/schema/7/2/sample_data/generate.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+import json
+import lsst.alert.packet as packet
+from datetime import datetime
+import pandas as pd
+
+with open("alert.json", "r") as f:
+    data = json.load(f)
+
+    data_time = datetime.now()
+
+    data['diaSource']['time_processed'] = pd.Timestamp(data_time)
+    data['diaObject']['validityStart'] = pd.Timestamp(data_time)
+
+schema = packet.SchemaRegistry.from_filesystem(schema_root="lsst.v7_1.alert").get_by_version("7.1")
```

Comment:
Should these be `lsst.v7_2.alert` and `7.2`? Probably the failing tests may be fixed by updating versions here and regenerating `fakeAlert`?